website/plugins/customWebpackConfig/index.js

```diff
@@ -24,6 +23,11 @@ module.exports = function customWebpackConfigPlugin(context, options) {
           }),
           new NodePolyfillPlugin({}),
         ],
+        module: {
+          rules: [
+            { test: /\.py$/, loader: 'raw-loader' }
```

Comment:
Is this basically making sure it loads Python files as raw data? Is this necessary in this PR if we're not actually loading in any Python?